gtsfm/utils/features.py

```diff
@@ -106,6 +101,25 @@ def compute_point_line_distances(points: np.ndarray, lines: np.ndarray) -> np.nd
     Returns:
         Point-line distance for each row, of shape N.
     """
-    line_norms = np.linalg.norm(lines[:, :2], axis=1)
+    line_norms = np.linalg.norm(lines[:, :2], axis=1) + EPS
```

Comment:
was this causing an issue (divide by zero)?

Reply:
I do not exactly recall where (ad if) I encountered this. But I guess it should be good to put it in there.